library/clinical_rules.php

```diff
@@ -1914,13 +1914,28 @@ function exist_database_item($patient_id,$table,$column='',$data_comp,$data='',$
       "WHERE " . add_escape_custom($patient_id_label)  . "=? " . $customSQL, array($patient_id) );
   }
   else {
-    // search for number of specific items
-    $sql = sqlStatementCdrEngine("SELECT `" . add_escape_custom($column) . "` " .
-      "FROM `" . add_escape_custom($table)  . "` " .
-      " ". $whereTables. " ".
-      "WHERE `" . add_escape_custom($column) ."`" . $compSql . "? " .
-      "AND " . add_escape_custom($patient_id_label)  . "=? " . $customSQL .
-      $dateSql, array($data,$patient_id) );
+      if ($whereTables=="" && strpos($table, 'form_')!== false){
```

Comment:
No , this handles only the specific case that a column is given 
and  that the table represent a form